src/WebInterface/Presentation/Http/Form/OpenType.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gaming\WebInterface\Presentation\Http\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Choice;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+final class OpenType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('size', ChoiceType::class, [
+                'label' => 'Size',
+                'label_attr' => ['class' => 'btn'],
+                'attr' => ['class' => 'btn-group w-100'],
+                'choices' => self::sizes(),
+                'choice_attr' => static fn() => ['class' => 'btn-check'],
+//                'expanded' => true,
```

Comment:
Needs to be enabled. However, acceptance tests (except the first, which uses the same functionality tho) in the pipeline fails, although acceptance tests are running locally (and the deployed page works too).